src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,60 @@
 package mate.academy.lib;
 
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class Injector {
     private static final Injector injector = new Injector();
 
+    private Map<Class<?>, Object> instances = new HashMap<>();
```

Comment:
Empty line
```suggestion
    private static final Injector injector = new Injector();
    private Map<Class<?>, Object> instances = new HashMap<>();
```

Reply:
static and nonstatic fields must be separated